dev-lang/r/r-3.3.2.recipe

```diff
@@ -0,0 +1,119 @@
+SUMMARY="A programming language primarily for statistics and numerical analysis"
+DESCRIPTION="R is GNU S, a system for statistical computation and graphics. \
+It consists of a language plus a run-time environment with graphics, a \
+debugger, access to certain system functions, and the ability to run programs \
+stored in script files. The core of R is an interpreted computer language \
+which allows branching and looping as well as modular programming using \
+functions. The R distribution contains functionality for a large number of \
+statistical procedures such as: linear and generalized linear models, \
+nonlinear regression models, time series analysis, classical parametric and \
+nonparametric tests, clustering, smoothing and graphical data \
+representations. Additional modules ('add-on packages') are available for a \
+variety of specific purposes but are not distributed with this package."
+HOMEPAGE="http://www.r-project.org/"
+COPYRIGHT="1993-2016 The R Core Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://cloud.r-project.org/src/base/R-3/R-$portVersion.tar.gz"
+CHECKSUM_SHA256="d294ad21e9f574fb4828ebb3a94b8cb34f4f304a41687a994be00dd41a4e514c"
+SOURCE_DIR="R-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	r$secondaryArchSuffix = $portVersion
+	cmd:r$secondaryArchSuffix
+	cmd:rscript$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	#lib:libcairo$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libgettextpo$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libicui18n$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libpaper$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libtcl8.5$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	#devel:libblas$secondaryArchSuffix
+	#devel:liblapack$secondaryArchSuffix
+	#devel:libcairo$secondaryArchSuffix
```

Comment:
Because i want to recompile them with fortran. It is a nice test for gfortran. But it takes time.